specification/workloads/resource-manager/Microsoft.Workloads/preview/2021-12-01-preview/monitors.json

```diff
@@ -286,7 +286,12 @@
             "$ref": "#/parameters/monitorNameParameter"
           },
           {
-            "$ref": "#/parameters/TagsParameter"
+            "in": "body",
+            "name": "body",
+            "description": "The Update SAP workload monitor request body.",
+            "schema": {
+              "$ref": "#/definitions/UpdateMonitorRequest"
+            }
```

Comment:
Please make this parameter `required: true`.
It makes no sense that the body parameter of a patch operation is optional. And in addition, in previous version, the body parameter is required. This impacts the SDKs too much.

Reply:
Made `required`